css-contain-2/Overview.bs

```diff
@@ -2308,14 +2339,7 @@ Restrictions and Clarifications {#cv-notes}
 	</wpt>
 
 3. If an element starts or stops [=skipped contents|skipping its contents=],
-	this change happens
-	after the requestAnimationFrame callbacks
-	of the frame that renders the effects of the change
```

Comment:
So there's a bit of a subtlety here. What this is saying is that if we have frames 1 and 2 (separate update the rendering steps), and we determine that an element is relevant in step 1, then the effect of that relevance (ie it allows rendering, doesn't have strict containment etc) happen on frame 2. The difference is that if you try and observe the size of the element between rAFs in a timeout, then between frame 1 and 2 it should act as if it's still not relevant wrt size containment. See the note in this spec below which tries to explain that.

I don't mind if we change this behavior, but we need a larger discussion for that.

Reply:
That's what the intention of the _currently relevant to the user_ flag is (maybe I botched the language).

"currently relevant to the user" is meant to be the thing that has effect on the rendering. But you're right I might have messed up my prose ;)

Reply:
Ok, I think 4248490afa1fbd774f5d385fb952afd4f4ba00d7 preserves the behavior you were talking about. Though I think I'd rather treat the initial and non-initial viewport determination the same, wdyt?